frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.jsx

```diff
@@ -78,6 +78,9 @@ export function SmartScalar({
   visualizationIsClickable,
   series: [
     {
+      card: {
+        dataset_query: { type: queryType },
+      },
```

Comment:
Matches this from the original PR - https://github.com/metabase/metabase/pull/38344/files#diff-195cb3e525a65af942309b028c271969b37c05448843666c178cb3708c266004R158-R160